tests/station07.test.tsx

```diff
@@ -1,6 +1,24 @@
-// No tests for Station No.7.
-// Proceed to the meeting reservation.
-// TODO: ここのテストを書く
-// TODO: クリアしてほしい基準を決定→たたきを大木へ送る(→問題文からみて決定する)
+import { render } from '@testing-library/react'
+import React from 'react'
```

Comment:
```suggestion
import * as React from 'react'
```

default importではなくmodule importにするとtree-shakingの恩恵を受けられるようになりますよ！

Reply:
station11の方も`import * as React from 'react'`に合わせたほうがいいですよね？

Reply:
そちらもお願いします！（別プルリクで作業することになるとは思いますが...）